### DISCUSSION TITLE
Bulljs with redis cluster mode?

### DISCUSSION THREAD

**Discussion by michgboxy2** [OptimalBits/bull#2067] [Q&A]
What is the drawback to connecting to an ElasticCache Redis cluster mode with the normal connection settings?
i.e const autoreselectShiftQueue = new Queue('auto-reselection', process.env[process.env.REDIS_URL_ENV_NAME]);

cos I was able to connect successfully with this, but my jobs get deleted irrespective of the job status after 15 minutes. whereas it doesn't happen on a local Redis. It only happens with a clustered Elasticcache Redis.

Does anyone know what the issue might be?

**Comment by manast** [comment]
I do not know about anything that would delete jobs after 15 minutes, cluster or otherwise. In any case it is simple to just test with non clustered elasticache to see if there is a difference.
